cli/src/commands/describe.rs

```diff
@@ -181,6 +183,19 @@ pub(crate) fn cmd_describe(
         })
         .collect_vec();
 
+    // micro optimization for the default empty trailer template: if the template
+    // is empty, we don't have to write temp commit. we can also avoid re-parsing
+    // template.
```

Comment:
nit: oops, I didn't mean this comment to be included.

Can you also restore the original comment why `use_editor || !is_empty()`?

Reply:
Thanks, forgot to put that comment back.